src/logger.js

```diff
@@ -1,9 +1,13 @@
 const winston = require('winston')
 const path = require('path')
 const DailyRotateFile = require('winston-daily-rotate-file')
-const store = require('./store')
 const util = require('util')
 
+const store = require('./store')
+const appVersion = require('../package.json').version
+
+const BUGSNAG_API_KEY = 'fcd92279c11ac971b4bd29b646ec4125'
```

Comment:
Will this let anybody make API requests to bugsnag? Could randos use this to read sensitive reports?

Reply:
This will let anyone make API requests to bugsnag. I'll hide it in the source code using an environment variable. It won't let randos read sensitive reports.